astroid/brain/brain_turtle.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) 2020 rioj7
+
+"""
+Run the following script to generate the string to parse in the transform method
+
+import turtle
+def _make_global_funcs(functions, cls):
+  funcs = []
+  for methodname in functions:
+    method = getattr(cls, methodname)
+    paramslist, argslist = turtle.getmethparlist(method)
+    if paramslist == "": continue
+    funcs.append(f"def {methodname}{paramslist}: return")
+  return funcs
+funcs = []
+funcs.extend(_make_global_funcs(turtle._tg_screen_functions, turtle._Screen))
+funcs.extend(_make_global_funcs(turtle._tg_turtle_functions, turtle.Turtle))
+print('\n'.join(funcs))
+"""
```

Comment:
It is interesting but i'm not sure it is the right place to store this script. Why not in a proper repo on your github account, with a link toward it?

Reply:
In fact i'v got a similar script that i used for `numpy` but it doesn't make part of `astroid` sources.